src/tools/schema2base.ts

```diff
@@ -15,6 +15,8 @@ import {Runtime} from '../runtime/runtime.js';
 import {SchemaGraph, SchemaNode} from './schema2graph.js';
 import {ParticleSpec} from '../runtime/particle-spec.js';
 
+Runtime.init('../..');
```

Comment:
Why move this to the top of the file from the ctor?

Reply:
Runtime init cannot be called multiple times, so you could not instantiate schema2kotlin more than once in a node process. This was causing a pain with these functions calculating outputs.